egg/zoo/dsprites_bvae/README.md

```diff
@@ -0,0 +1,8 @@
+[beta-VAE](https://openreview.net/pdf?id=Sy2fzU9gl) as a communication game between Sender (Encoder) and Receiver (Decoder). 
+The game can be used to play around with continuous channel and visual input. By default, topographic similarity and positional disentanglement are logged.
+```bash
+cd EGG
+pip install wheel editdistance;
```

Comment:
editdistance shouldn't be needed as it's already a dependency for EGG https://github.com/facebookresearch/EGG/blob/master/requirements.txt#L6